source/vocab/display.jsonld

```diff
@@ -134,6 +134,11 @@
           "@type": "fresnel:Lens",
           "classLensDomain": "Contribution",
           "showProperties": [ "agent", "role" ]
+        },
+        "Relationship": {
+          "@type": "fresnel:Lens",
+          "classLensDomain": "Relationship",
+          "showProperties": [ "entity" ]
```

Comment:
We should add `"relation"` to `"showProperties"` too (it's related to `role` but used for document relationships).

Reply:
I was thinking about this, but couldn't wrap my head around how that would look. Maybe we should just add that one too and see :)

Reply:
Go for it! :)